static/assets/js/vendors/base64.js

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2017 Duo Security, Inc. All rights reserved.
```

Comment:
same question for this one, can it be installed via `npm`?

Reply:
Same as [this](https://github.com/simple-login/app/pull/159#discussion_r421366256)